docs/source/dev_guide/algorithms/k_nearest_neighbors/k-nearest-neighbors-knn-classifier.rst

```diff
@@ -56,10 +53,21 @@ Given a positive integer parameter :math:`k` and a test observation
 #. Assigns the class with the largest probability to the test
    observation :math:`x_0`
 
-|product| version of the kNN algorithm uses the PANDA algorithm
-[Patwary2016]
-that uses a space-partitioning data structure known as K-D tree. Each
-non-leaf node of a tree contains the identifier of a feature along
+The library provides kNN classification with the following methods:
```

Comment:
could they be called search techniques?

Reply:
As quick fix you can use use 'approaches' term, but actually it's about usage of the index for k-d tree case, and approach without any index for brute-force case. I suggest to consult author of those changes, because it's his responsibility to formulate it in the correct way.

Reply:
Ok, used "approaches" for now